packages/framework/tests/Unit/Component/UploadedFile/UploadedFileLocatorTest.php

```diff
@@ -81,24 +81,6 @@ public function testGetRelativeUploadedFileFilepath()
         );
     }
 
-    public function testGetUploadedFileUrl()
```

Comment:
Why is this test removed? I do not see the test which covers returning URL to uploaded file. Maybe I am blind.

Reply:
Because method right now only generates URL using `DomainRouter` and we already have test for that.

Reply:
Where can I find the test? It is not in `UploadedFileLocatorTest`.

Reply:
CurrentDomainRouterTest